examples/forward-camera/README.md

```diff
@@ -0,0 +1 @@
+A platformer camera that leans towards the player's direction. Making the obstacles more visible
```

Comment:
"A platformer camera that leans toward the direction the player is looking at. Useful to give more visibility to the player when running through a level."

(careful about your original description, it does not include a period at the end, and you can't start a sentence with "Making the [...]")